tardis/montecarlo/packet_source.py

```diff
@@ -129,24 +132,26 @@ class BlackBodySimpleSource(BasePacketSource):
     def create_packets(self, temperature, no_of_packets, radius):
         """Generate black-body packet properties as arrays
 
-        Parameters
-        ----------
-        temperature : float64
-        no_of_packets : int
-            Number of packets
-        radius : float64
-            Initial packet radius
-
-        Returns
-        -------
-        array
-            Packet radii
-        array
-            Packet frequencies
-        array
-            Packet directions
-        array
-            Packet energies
+                Parameters
+                ----------
+                temperature : astropy.Quantity
+                    Absolute Temperature
+                no_of_packets : int
+                    Number of packets
+        <<<<<<< HEAD
+                radius : float64
+        =======
+                rng : numpy random number generator
+                radius : astropy.Quantity
+        >>>>>>> a0ca0e139 (Add units to create_packets)
+                    Initial packet radius
+
```

Comment:
This was probably a merge conflict being resolved but was staged.

Reply:
Resolved this!